src/test/java/app/coronawarn/verification/VerificationApplicationTest.java

```diff
@@ -75,6 +46,20 @@
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.web.servlet.MockMvc;
+
+import java.io.UnsupportedEncodingException;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.PrivateKey;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.*;
```

Comment:
same like here

Reply:
I've rearranged the imports for this file